app/src/main/java/com/yacgroup/yacguide/AboutActivity.kt

```diff
@@ -36,12 +38,33 @@ class AboutActivity : BaseNavigationActivity() {
         super.onCreate(savedInstanceState)
         this.title = getString(R.string.menu_about, getString(R.string.app_name))
 
-        val textViewAppVersion: TextView = findViewById(R.id.textViewAppVersion)
-        textViewAppVersion.text = "Version ${getAppVersion()}"
-        textViewAppVersion.gravity = Gravity.CENTER_HORIZONTAL
+        _createEntry(getString(R.string.data_src), getString(R.string.data_src_url),null)
+        _createEntry(getString(R.string.privacy_policy), null) { _showPrivacyPolicy() }
+        _createEntry(
+                getString(R.string.source_code_and_support),
+                getString(R.string.github_url),
+                null )
+        _createEntry(getString(R.string.license), getString(R.string.license_url),null)
+        _createEntry(getString(R.string.version), _getAppVersion(), null)
     }
 
-    private fun getAppVersion() : String {
+    private fun _createEntry(
+            title: String,
+            description: String?,
+            callback: ((view: View) -> Unit)?) {
+        val contentAbout: LinearLayout = findViewById(R.id.aboutContent)
+        val entry = layoutInflater.inflate(R.layout.about_entry, null)
+        if (callback == null && URLUtil.isValidUrl(description)) {
+            entry.setOnClickListener { description?.let { url -> _openUrl(url) } }
+        } else {
+            entry.setOnClickListener(callback)
```

Comment:
This will be also triggered if `callback == null` and `description` is not a valid url. Might lead to NullPointerException